src/ic-websocket-gateway/src/client_session_handler.rs

```diff
@@ -180,12 +185,28 @@ impl ClientSessionHandler {
                 Ok(Some(IcWsSessionState::Open)) => {
                     client_session_span.in_scope(|| {
                         debug!("Client session opened");
+
+                        gauge!("clients_connected").increment(1.0);
+
+                        let client_key = self.get_client_key(&client_session);
+                        let key = format!("client_{}", client_key.clone().to_string());
```

Comment:
@AvoidMelon what's the reason of prepending `client_` to the client_key? Can't we just use the client key as the HashMap's key directly?